src/compression.rs

```diff
@@ -218,14 +222,14 @@ pub fn create_encoding_header(existing: Option<HeaderValue>, coding: ContentCodi
     coding.into()
 }
 
-/// Try to get the prefered `content-encoding` via the `accept-encoding` header.
+/// Try to get the preferred `content-encoding` via the `accept-encoding` header.
 #[inline(always)]
-pub fn get_prefered_encoding(headers: &HeaderMap<HeaderValue>) -> Option<ContentCoding> {
+pub fn get_preferred_encoding(headers: &HeaderMap<HeaderValue>) -> Option<ContentCoding> {
```

Comment:
Comment: Unfortunately, this function is public. So we should mention this breaking change in the next future release.